### ISSUE TITLE
quadrupled Footer

### ISSUE THREAD

**Issue by MartiPau** [qtranslate/qtranslate-xt#846]
Good afternoon,
web: https://vidaltech.net/
When updating my template impreza to 7.5 and my wordpress to 5.4.1 the footer has been quadrupled. One for each language. I know that you don't offer help for the qtranslate plugin but I don't know if the problem is due to this plugin.
Is there a possibility to downgrade the version of the impreza template?
Thank you very much in advance,
Mar

Versions:
{
  "PHP_VERSION": "7.3.18",
  "WP_VERSION": "5.4.1",
  "QTX_VERSION": "3.8.1",
  "Plugins": [
    "Ultimate Addons for WPBakery Page Builder 3.18.0",
    "Akismet Anti-Spam 4.1.5",
    "Asesor de Cookies RGPD 0.31",
    "iThemes Security 7.7.1",
    "Editor clásico 1.5",
    "Contact Form 7 5.1.9",
    "Contact Form CFDB7 1.2.4.8",
    "Force Regenerate Thumbnails 2.0.6",
    "Google Analytics Dashboard for WP (GADWP) 6.0.2",
    "WPBakery Visual Composer &amp; qTranslate-X 1.0",
    "WPBakery Page Builder 6.2.0",
    "Loco Translate 2.4.0",
    "Post Types Order 1.9.4.3",
    "qTranslate-XT 3.8.1",
    "Really Simple SSL 3.3.3",
    "Slider Revolution 6.2.9",
    "SEO Ultimate 7.6.5.9",
    "UpSolution Core 7.5.2",
    "Visual Composer Add-on Image Hotspot with Tooltip 1.5.1",
    "WP Downgrade | Specific Core Version 1.2.1"
  ]
}

Configuration
{
  "default_language": "es",
  "enabled_languages": [
    "es",
    "ca",
    "en",
    "fr"
  ],
  "flag_location": "plugins/qtranslate-xt-3.8.1/flags/",
  "language_name": {
    "es": "Español",
    "ca": "Català",
    "en": "English",
    "fr": "Français"
  },
  "locale": {
    "es": "es_ES",
    "ca": "ca",
    "en": "en_US",
    "fr": "fr_FR"
  },
  "locale_html": {
    "es": "",
    "ca": "",
    "en": "",
    "fr": ""
  },
  "not_available": {
    "es": "Disculpa, pero esta entrada está disponible sólo en %LANG:, : y %.",
    "ca": "Ho sentim, aquesta entrada es troba disponible únicament en %LANG:, : i %.",
    "en": "Sorry, this entry is only available in %LANG:, : and %.",
    "fr": "Désolé, cet article est seulement disponible en %LANG:, : et %."
  },
  "date_format": {
    "es": "%d \\d\\e %B \\d\\e %Y",
    "ca": "j F, Y",
    "en": "%A %B %e%q, %Y",
    "fr": "%A %e %B %Y"
  },
  "time_format": {
    "es": "%H:%M hrs.",
    "ca": "G:i",
    "en": "%I:%M %p",
    "fr": "%H:%M"
  },
  "flag": {
    "es": "es.png",
    "ca": "catala.png",
    "en": "gb.png",
    "fr": "fr.png"
  },
  "url_mode": 2,
  "use_strftime": 3,
  "filter_options_mode": 0,
  "language_name_case": 0,
  "detect_browser_language": true,
  "hide_untranslated": false,
  "show_displayed_language_prefix": false,
  "show_alternative_content": false,
  "hide_default_language": true,
  "use_secure_cookie": false,
  "header_css_on": true,
  "header_css": ".qtranxs_flag_es {background-image: url(https://vidaltech.net/wp-content/plugins/qtranslate-xt-3.8.1/flags/es.png); background-repeat: no-repeat;}\n.qtranxs_flag_ca {background-image: url(https://vidaltech.net/wp-content/plugins/qtranslate-xt-3.8.1/flags/catala.png); background-repeat: no-repeat;}\n.qtranxs_flag_en {background-image: url(https://vidaltech.net/wp-content/plugins/qtranslate-xt-3.8.1/flags/gb.png); background-repeat: no-repeat;}\n.qtranxs_flag_fr {background-image: url(https://vidaltech.net/wp-content/plugins/qtranslate-xt-3.8.1/flags/fr.png); background-repeat: no-repeat;}\n",
  "text_field_filters": [],
  "front_config": {
    "all-pages": {
      "filters": {
        "text": {
          "widget_title": "20",
          "widget_text": "20",
          "the_title": "20",
          "category_description": "20",
          "list_cats": "20",
          "wp_dropdown_cats": "20",
          "term_name": "20",
          "get_comment_author": "20",
          "the_author": "20",
          "tml_title": "20",
          "term_description": "20"
        },
        "term": {
          "cat_row": "0",
          "cat_rows": "0",
          "wp_get_object_terms": "0",
          "single_cat_title": "0",
          "single_tag_title": "0",
          "single_term_title": "0",
          "the_category": "0",
          "get_term": "0",
          "get_terms": "0",
          "get_category": "0"
        },
        "url": {
          "author_feed_link": "10",
          "author_link": "10",
          "get_comment_author_url_link": "10",
          "post_comments_feed_link": "10",
          "day_link": "10",
          "month_link": "10",
          "year_link": "10",
          "page_link": "10",
          "post_link": "10",
          "category_link": "10",
          "category_feed_link": "10",
          "tag_link": "10",
          "term_link": "10",
          "the_permalink": "10",
          "feed_link": "10",
          "tag_feed_link": "10"
        }
      }
    }
  },
  "term_name": {
    "Sector ferroviari": {
      "ca": "Sector ferroviari",
      "es": "Sector ferroviario",
      "en": "Railway sector",
      "fr": "Secteur ferroviaire"
    },
    "Enginyeries i projectes": {
      "ca": "Enginyeries i projectes",
      "es": "Ingenierias y proyectos",
      "en": "Engineering and projects",
      "fr": "Ingénieries et projets"
    },
    "Fabricants de maquinaria": {
      "ca": "Fabricants de maquinaria",
      "es": "Fabricantes de maquinaria",
      "en": "Machine manufacturers",
      "fr": "Fabricants de machine"
    }
  },
  "disable_client_cookies": false,
  "ignore_file_types": [
    "gif",
    "jpg",
    "jpeg",
    "png",
    "svg",
    "pdf",
    "swf",
    "tif",
    "rar",
    "zip",
    "7z",
    "mpg",
    "divx",
    "mpeg",
    "avi",
    "css",
    "js",
    "mp3",
    "mp4",
    "apk"
  ],
  "url_info": {
    "cookie_enabled": true,
    "cookie_lang_front": "ca",
    "cookie_lang_admin": "es",
    "scheme": "https",
    "host": "vidaltech.net",
    "path": "/wp-admin/admin-ajax.php",
    "query": "action=admin_debug_info",
    "path-base": "",
    "path-base-length": 0,
    "wp-path": "/wp-admin/admin-ajax.php",
    "language_neutral_path": true,
    "http_referer": "https://vidaltech.net/wp-admin/options-general.php?page=qtranslate-xt",
    "referer_admin": true,
    "doing_front_end": false,
    "lang_cookie_admin": "es",
    "lang_admin": "es",
    "language": "es",
    "set_cookie": false,
    "page_referer": "options-general.php"
  },
  "language": "es",
  "editor_mode": 0,
  "highlight_mode": 1,
  "auto_update_mo": true,
  "hide_lsb_copy_content": false,
  "lsb_style": "Simple_Buttons.css",
  "config_files": [
    "./i18n-config.json"
  ],
  "admin_config": {
    "post": {
      "pages": {
        "post.php": "",
        "post-new.php": ""
      },
      "anchors": {
        "post-body-content": {
          "where": "first last"
        }
      },
      "forms": {
        "post": {
          "fields": {
            "title": [],
            "excerpt": [],
            "attachment_caption": [],
            "attachment_alt": [],
            "view-post-btn": {
              "encode": "display"
            },
            "wp-editor-area": {
              "jquery": ".wp-editor-area"
            },
            "gallery-caption": {
              "jquery": ".gallery-caption",
              "encode": "none"
            },
            "wp-caption-text": {
              "jquery": ".wp-caption-text",
              "encode": "display"
            },
            "img-alt": {
              "jquery": "img",
              "attrs": [
                "alt"
              ],
              "encode": "display"
            }
          }
        },
        "wpbody-content": {
          "fields": {
            "wrap-h1": {
              "jquery": ".wrap h1",
              "encode": "display"
            },
            "wrap-h2": {
              "jquery": ".wrap h2",
              "encode": "display"
            }
          }
        }
      },
      "js-exec": {
        "post-exec": {
          "src": "./admin/js/post-exec.min.js"
        }
      }
    },
    "options-general": {
      "preg_delimiter": "#",
      "pages": {
        "options-general.php": "^(?!.*page=[^=&]+).*$"
      },
      "forms": {
        "options": {
          "fields": {
            "blogname": [],
            "blogdescription": [],
            "head-title": {
              "jquery": "head title",
              "encode": "display"
            }
          }
        }
      }
    },
    "widgets": {
      "pages": {
        "widgets.php": ""
      },
      "anchors": {
        "widgets-right": {
          "where": "before after"
        }
      },
      "forms": {
        "widgets-right": {
          "fields": {
            "widget-title": {
              "jquery": "input[id^='widget-'][id$='-title']"
            },
            "widget-text-text": {
              "jquery": "textarea[id^='widget-text-'][id$='-text']"
            },
            "in-widget-title": {
              "jquery": "span.in-widget-title",
              "encode": "display"
            }
          }
        }
      },
      "js-exec": {
        "widgets-exec": {
          "src": "./admin/js/widgets-exec.min.js"
        }
      }
    },
    "edit-term": {
      "pages": {
        "term.php": "",
        "edit-tags.php": "action=edit"
      },
      "forms": {
        "edittag": {
          "fields": {
            "name": [],
            "description": [],
            "parent": {
              "encode": "display"
            }
          }
        }
      }
    },
    "edit-tags": {
      "preg_delimiter": "#",
      "pages": {
        "edit-tags.php": "^(?!.*action=edit).*$"
      },
      "anchors": {
        "posts-filter": {
          "where": "before after"
        }
      },
      "forms": {
        "addtag": {
          "fields": {
            "tag-name": [],
            "tag-description": [],
            "parent": {
              "encode": "display"
            }
          }
        },
        "col-left": {
          "fields": {
            "tagcloud": {
              "jquery": ".tagcloud > a",
              "encode": "display"
            }
          }
        },
        "the-list": {
          "fields": {
            "row-title": {
              "jquery": ".row-title",
              "encode": "display"
            },
            "description": {
              "jquery": ".description",
              "encode": "display"
            }
          }
        }
      },
      "js-exec": {
        "edit-tags-exec": {
          "src": "./admin/js/edit-tags-exec.min.js"
        }
      }
    },
    "nav-menus": {
      "pages": {
        "nav-menus.php": "action=edit|menu=\\d+|^$"
      },
      "anchors": {
        "menu-to-edit": {
          "where": "before after"
        }
      },
      "forms": {
        "update-nav-menu": {
          "fields": {
            "title": {
              "jquery": "[id^=edit-menu-item-title-]"
            },
            "attr-title": {
              "jquery": "[id^=edit-menu-item-attr-title-]"
            },
            "description": {
              "jquery": "[id^=edit-menu-item-description-]"
            },
            "span.title": {
              "jquery": "span.menu-item-title",
              "encode": "display"
            },
            "link-to-original": {
              "jquery": ".link-to-original >a",
              "encode": "display"
            }
          }
        },
        "side-sortables": {
          "fields": {
            "label.title": {
              "jquery": "label.menu-item-title",
              "encode": "display"
            },
            "accordion-section-title": {
              "jquery": "h3.accordion-section-title",
              "encode": "display"
            }
          }
        }
      },
      "js-exec": {
        "nav-menus-exec": {
          "src": "./admin/js/nav-menus-exec.min.js"
        }
      }
    },
    "all-pages": {
      "filters": {
        "text": {
          "admin_title": "20"
        }
      }
    }
  },
  "custom_i18n_config": [],
  "custom_fields": [],
  "custom_field_classes": [],
  "post_type_excluded": [],
  "translator": {},
  "qtrans_compatibility": false,
  "i18n-cache": {
    "admin_page_configs": {
      "": {
        "filters": {
          "text": {
            "admin_title": "20"
          }
        }
      }
    }
  }
}

**Comment by herrvigg** [comment]
> the footer has been quadrupled. One for each language. 

What do you mean with this? I don't see any problem on your website.

I can only see you have a `[:es][:ca][:en]` string appearing in the footer (HTML), that looks like a leftover or something strange. If you don't want the footer translated, you can remove all translation tags and it will be displayed for any language. But i don't know how this footer is supposed to be translated.

> Is there a possibility to downgrade the version of the impreza template?

Sorry i have no clue what this template is about, this does not concern qTranslate.

**Comment by MartiPau** [comment]
Hello herrvigg, first of all thank you for replying :)
He had replaced the footer with another, removing the translations so that it would not be duplicated for users.
Now I just activated the original footer with the translations.

The strange thing is that the rest of the web is translated correctly and it is the footer that is duplicated by language.

Any idea what can happen?

Thank you,

**Comment by herrvigg** [comment]
Mmmm it depends a bit how this footer is generated. If it's not standard WordPress that could explain why, because you are missing some hooks that would perform the translation of that part (for the front-end side).

**Comment by herrvigg** [comment]
@MartiPau any update? I don't really see how this footer is generated so it is hard to know what to do or where to look.

**Comment by herrvigg** [comment]
Without information to investigate, impossible to understand the problem.
